### ISSUE TITLE
CI for the docs

### ISSUE THREAD

**Issue by fabianmoronzirfas** [basiljs/basiljs.github.io#11]

**Comment by fabianmoronzirfas** [comment]
Circle CI looks most promessing

**Comment by fabianmoronzirfas** [comment]
Did a little test with travis-ci https://github.com/fabianmoronzirfas/automate-build 

- it can build jekyll or custom github pages
- it can trigger a node script on commit
- it can't be triggerd when a commit is done on a different repo

I'm wondering what the strategy could be?  

- trigger a build using cronjobs?
- trigger builds manually?
- do we need CI for the docs?

Any suggestions? @basiljs/basil-js-core-members

**Comment by fabianmoronzirfas** [comment]
Did some testing with github actions. I can trigger a npm run … from a push to another repo. 🤖 

Things i need for this:

- a user that does that. I will call him basilbot
- github actions to be out of beta and available to the public

I'll keep you posted and will document the process asap.

**Comment by fabianmoronzirfas** [comment]
I found a possibility to update our docs whenever we do a push the main basil repo.
The gist of it can be found here https://fabianmoronzirfas.me/tech/2019/02/06/cross-repo-github-action-triggering.html

TLDR   

We have to wait until GitHubs Actions feature is out of beta. Then we can have some webhooks on the one trigger a build and commit on the other one. 🤖🚀
